R/map_fetch.R

```diff
@@ -201,8 +206,9 @@ map_fetch <- function(
       year <- paste0(c(min(year), max(year)), collapse = ",")
     }
   }
-
-  query <- rgbif_compact(list(srs = srs, taxonKey = taxonKey,
+  
+  input_srs <- ifelse(grepl("mvt", format), "EPSG:3857", srs)
```

Comment:
@jeroen problem is i think that here we're forcing srs to be 3857 if mvt is requested, but then below `srs` is used when passing to `protolite::read_mvt_sf` - so you always request 3857 from GBIF, but then protolite uses whatever the user passed in. Is that what you intended?

Reply:
Yes, protolite only understands `3857` input data, so we always need that. And then we ask `protolite::read_mvt_sf` to convert it to whatever output the user requested.

Reply:
okay, i think we should then only allow `3857` when `format="mvt"`, correct?

Reply:
Yes only allow `"EPSG:3857"` as input data from rgbif. The output crs may be be anything.

Reply:
ah, so the user can still set a different srs for the output, the srs that's passed to `protolite::read_mvt_sf`?

Reply:
Indeed, in case of mvt, the input data is always 3857 but we can transform it into the desired coordinate system after parsing it.